### PR TITLE
The existing python script has been updated (with a new name) to impl…

### PR DESCRIPTION
**Pull request by adhidevi**
…iment the latest showermax geometry. The light guide and quartz-tungsten stack have new dimensions. Each quartz is now 6 mm thick and the quartz-tungsten stack has been shifted upstream by a distance equivalent to the thickness of a tungsten tile so that the opening of the light guide near the quartz covers only the four quartz and three interleaved tungsten layers. Lightguide thickness (as well as thickness of the mirror box for quartz-tungsten stack) has been changed to 0.5 mm. Quarter-inch thick aluminum plates have been added in the front and back of the quartz-tungsten stack.

### CODE REVIEW

**Comment by wdconinc** [comment]
Thanks for the updated code. Adding `_new` to files is not a sustainable practice. Can you apply the changes to the actual script?

**Comment by adhidevi** [comment]
Yes, I will make the changes in the original script and comment. Thank you.

File: showermaxgen.py [hunk 1]
```diff
@@ -290,11 +286,11 @@
         theta=math.pi+2*i*math.pi/28
         xpos=rpos*(math.cos(theta))
         ypos=rpos*(math.sin(theta)) 
-        if (i%2==0):	
-		zpos=-zstagger
+        if (i%2==0):    
+                zpos=zstagger
```
Comment (wdconinc) [inline showermaxgen.py]: Did the stagger change intentionally?

**Review by wdconinc — APPROVED** [review]

File: showermaxgen.py (hunk 1 above)
Comment (adhidevi) [reply showermaxgen.py]: No, sorry. I was checking the geometry by switching the positions of open/closed and transition modules. I thought I reverted it back. I will revert back the sign and commit. Thank you.

**Comment by wdconinc** [comment]
This change has instroduced overlaps.
image

Image for only 1 detector, `range(0,1)` instead of `range(0,28)`, and obtained with [gdmlview](https://github.com/JeffersonLab/gdmlview), `gdmlview -o showerMaxGen.gdml`

Red is overlapping volume. Yellow is the actual overlap. (not always easy to see....)

Where the red turns into yellow is where the top mirror box goes outside of the mother volume, for example.

```
-------- WWWW ------- G4Exception-START -------- WWWW -------
*** G4Exception : GeomVol1002
      issued by : G4PVPlacement::CheckOverlaps()
Overlap with mother volume !
          Overlap is detected for volume mirror_box_bot_0:0 (G4SubtractionSolid)
          with its mother volume logic_singledet_0 (G4UnionSolid)
          at mother local point (106.314,56.7348,35.4158), overlapping by at least: 3.39907 mm 
NOTE: Reached maximum fixed number -1- of overlaps reports for this volume !
*** This is just a warning message. ***
-------- WWWW -------- G4Exception-END --------- WWWW -------


-------- WWWW ------- G4Exception-START -------- WWWW -------
*** G4Exception : GeomVol1002
      issued by : G4PVPlacement::CheckOverlaps()
Overlap with mother volume !
          Overlap is detected for volume mirror_box_top_0:0 (G4SubtractionSolid)
          with its mother volume logic_singledet_0 (G4UnionSolid)
          at mother local point (152.454,-126.924,46.6953), overlapping by at least: 3.26128 mm 
NOTE: Reached maximum fixed number -1- of overlaps reports for this volume !
*** This is just a warning message. ***
-------- WWWW -------- G4Exception-END --------- WWWW -------


-------- WWWW ------- G4Exception-START -------- WWWW -------
*** G4Exception : GeomVol1002
      issued by : G4PVPlacement::CheckOverlaps()
Overlap with mother volume !
          Overlap is detected for volume front_plate_0:0 (G4Box)
          with its mother volume logic_singledet_0 (G4UnionSolid)
          at mother local point (-91.6965,-125.185,-28.5), overlapping by at least: 1.16965 cm 
NOTE: Reached maximum fixed number -1- of overlaps reports for this volume !
*** This is just a warning message. ***
-------- WWWW -------- G4Exception-END --------- WWWW -------


-------- WWWW ------- G4Exception-START -------- WWWW -------
*** G4Exception : GeomVol1002
      issued by : G4PVPlacement::CheckOverlaps()
Overlap with mother volume !
          Overlap is detected for volume back_plate_0:0 (G4Box)
          with its mother volume logic_singledet_0 (G4UnionSolid)
          at mother local point (42.9186,-131.575,34.85), overlapping by at least: 6.35 mm 
NOTE: Reached maximum fixed number -1- of overlaps reports for this volume !
*** This is just a warning message. ***
-------- WWWW -------- G4Exception-END --------- WWWW -------


-------- WWWW ------- G4Exception-START -------- WWWW -------
*** G4Exception : GeomVol1002
      issued by : G4PVPlacement::CheckOverlaps()
Overlap with mother volume !
          Overlap is detected for volume quartz_0_0:0 (G4ExtrudedSolid)
          with its mother volume logic_singledet_0 (G4UnionSolid)
          at mother local point (84.9374,-79.6898,26.9374), overlapping by at least: 1.01255 mm 
NOTE: Reached maximum fixed number -1- of overlaps reports for this volume !
*** This is just a warning message. ***
-------- WWWW -------- G4Exception-END --------- WWWW -------
```

File: showermaxgen.py [hunk 2]
```diff
@@ -156,28 +149,21 @@
 out+="\n\t\t<rotation name=\"rot_logic_mirror_box\" x=\"0\" y=\"0\" z=\"0\"/>"
 out+="\n\t</union>\n"
 
-
 out+="\t<union name=\"solid_logic_mirror_box_union\">"
```
Comment (wdconinc) [inline showermaxgen.py:164]: I think `solid_logic_mirror_box_union` needs another union to be big enough for the front and back plates, and `solid_logic_mirror_box` needs to be shifted by half a tungsten layer thickness.

**Review by wdconinc — CHANGES_REQUESTED** [review]

**Comment by wdconinc** [comment]
Thanks, no more overlaps now. I'll merge this tomorrow, when I can also put the showerMaxGen.gdml into remoll/geometry.

**Pull request merged**